src/main/java/org/killbill/billing/plugin/adyen/client/payment/converter/impl/NullObjectConverter.java

```diff
@@ -25,7 +25,7 @@
 public class NullObjectConverter implements PaymentInfoConverter {
 
     @Override
-    public PaymentType getPaymentType() {
+    public boolean supportsPaymentType(PaymentType type) {
         throw new UnsupportedOperationException("method is not supported!");
```

Comment:
Yup, it's mainly for debugging if we end-up using that converter for some reason (see [here](https://github.com/killbill/killbill-adyen-plugin/blob/d45e3c060373ac12e53c76b89d919573ea7d7d7f/src/main/java/org/killbill/billing/plugin/adyen/client/payment/builder/PaymentRequestBuilder.java#L68)).